library/std/src/f32/tests.rs

```diff
@@ -409,6 +409,22 @@ fn test_mul_add() {
     assert_eq!((-3.2f32).mul_add(2.4, neg_inf), neg_inf);
 }
 
+#[test]
+fn test_mul_add_fast() {
+    let nan: f32 = f32::NAN;
+    let inf: f32 = f32::INFINITY;
+    let neg_inf: f32 = f32::NEG_INFINITY;
+    assert_approx_eq!(12.3f32.mul_add_fast(4.5, 6.7), 62.05);
+    assert_approx_eq!((-12.3f32).mul_add_fast(-4.5, -6.7), 48.65);
+    assert_approx_eq!(0.0f32.mul_add_fast(8.9, 1.2), 1.2);
+    assert_approx_eq!(3.4f32.mul_add_fast(-0.0, 5.6), 5.6);
+    assert!(nan.mul_add_fast(7.8, 9.0).is_nan());
+    assert_eq!(inf.mul_add_fast(7.8, 9.0), inf);
+    assert_eq!(neg_inf.mul_add_fast(7.8, 9.0), neg_inf);
+    assert_eq!(8.9f32.mul_add_fast(inf, 3.2), inf);
+    assert_eq!((-3.2f32).mul_add_fast(2.4, neg_inf), neg_inf);
+}
```

Comment:
I believe this test is being compiled with the bootstrap stdlib, but these names don't exist in the bootstrap stdlib yet, so it's failing.
```suggestion
#[cfg(not(bootstrap))]
#[test]
fn test_mul_add_fast() {
    let nan: f32 = f32::NAN;
    let inf: f32 = f32::INFINITY;
    let neg_inf: f32 = f32::NEG_INFINITY;
    assert_approx_eq!(12.3f32.mul_add_fast(4.5, 6.7), 62.05);
    assert_approx_eq!((-12.3f32).mul_add_fast(-4.5, -6.7), 48.65);
    assert_approx_eq!(0.0f32.mul_add_fast(8.9, 1.2), 1.2);
    assert_approx_eq!(3.4f32.mul_add_fast(-0.0, 5.6), 5.6);
    assert!(nan.mul_add_fast(7.8, 9.0).is_nan());
    assert_eq!(inf.mul_add_fast(7.8, 9.0), inf);
    assert_eq!(neg_inf.mul_add_fast(7.8, 9.0), neg_inf);
    assert_eq!(8.9f32.mul_add_fast(inf, 3.2), inf);
    assert_eq!((-3.2f32).mul_add_fast(2.4, neg_inf), neg_inf);
}
```